scripts/tests

```diff
@@ -582,6 +582,12 @@ class Tests:
             l = TEST_TYPES.examples()
         else:
             l = list(map(TEST_TYPES.from_string, self.args.l))
+
+        # Filter out excluded examples
+        if hasattr(self.args, "exclude_example") and self.args.exclude_example:
```

Comment:
This `hasattr(self.args, "exclude_example")` seems unnecessary as there's a `default`. I think one can just drop the conditional.

Reply:
This was copied from mlkem-native. I prefer to keep it aligned and it does not bother me enough that I want to change it in mlkem-native.